package.json

```diff
@@ -77,16 +77,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.0",
+    "@angular/animations": "^11.2.5",
     "@angular/cdk": "^8.2.3",
```

Comment:
https://www.npmjs.com/package/@angular/cdk
This is angular material.. wondering why we use that. But most current version of this is `11.2.4` - seems like they try to match the angular version

Reply:
@flemcito It's the CDK of Angular Material - not the entire MD.
It was added for - reasons... 🤦🏻 😉 
We'll probably get rid of it when moving to Stencil (and/or re-introduce it with the Virtual Scrolling bug....)
But yes @jkaltoft the versions should be aligned with @angular/core etc.